configure

```diff
@@ -773,7 +773,13 @@ def glob_to_var(dir_base, dir_sub):
     (path, dirs, files) = ent
     for file in files:
       if file.endswith('.cpp') or file.endswith('.c') or file.endswith('.h'):
-        list.append('%s/%s' % (dir_sub, file))
+        srcfile = '%s/%s' % (dir_sub, file)
+        if patch_dir:
+          patchfile = '%s/%s/%s' % (dir_base, patch_dir, file)
+          if os.path.isfile(patchfile):
+            srcfile = '../../%s' % patchfile
```

Comment:
@orangemocha yes that is correct, these are paths for `gyp` relative to `tools/icu`. I think you are right about `patch_dir`, I will try that. thanks!